apps/react/src/features/genres/pages/GenresPage/GenresPage.tsx

```diff
@@ -20,14 +25,21 @@ const GenresPageComponent: FC = () => {
 	}
 
 	return (
-		<>
-			<h1>Genres</h1>
-			{genres.map(genre => <GenreCard key={genre.id} genre={genre} />)}
-		</>
+		<main className={styles.main}>
+			<Box className={styles.main__section}>
+				<GenresFilterForm/>
+				<Box className={styles.main__section__list}>
+					<GenresList genres={genres} />
+				</Box>
```

Comment:
Do we really need this wrapper? Can we do that?
```suggestion
				<GenresList genres={genres}  className={styles.main__section__list}/>
				
```